addons/l10n_ec/data/account_chart_template_setup_accounts.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+	<data noupdate="1">
+
```

Comment:
indent should be 4 spaces, not 8 or something

Reply:
Resolved